_data/projects/graspologic.yml

```
@@ -0,0 +1,13 @@
+name: graspologic
+desc: A Python package for network statistics and machine learning
+site: https://github.com/microsoft/graspologic
+tags:
+  - graph
+  - data-science
+  - machine-learning
+  - python
+  - networks
+  - statistics
+upforgrabs:
+  name: good first issue
+  link: https://github.com/microsoft/graspologic/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
```

Some tidy up to make this link more minimal:

```suggestion
  link: https://github.com/microsoft/graspologic/labels/good%20first%20issue
```